reference/import_export/connectors.rst

```diff
@@ -0,0 +1,16 @@
+XLSX connector
```

Comment:
Is the filename smart ? I figured we would add all future connector description to this file.